src/Directory.Packages.props

```diff
@@ -13,24 +13,24 @@
   <!-- Versions from OpenTelemetry.AutoInstrumentation.csproj -->
   <ItemGroup>
     <PackageVersion Include="MongoDB.Driver.Core.Extensions.DiagnosticSources" Version="1.4.0" />
-    <PackageVersion Include="OpenTelemetry.Exporter.Prometheus.HttpListener" Version="1.7.0-rc.1" />
-    <PackageVersion Include="OpenTelemetry.Exporter.Zipkin" Version="1.7.0" />
-    <PackageVersion Include="OpenTelemetry.Extensions.Propagators" Version="1.7.0" />
-    <PackageVersion Include="OpenTelemetry.Instrumentation.AspNet" Version="1.7.0-beta.2" />
-    <PackageVersion Include="OpenTelemetry.Instrumentation.AspNetCore" Version="1.7.1" />
-    <PackageVersion Include="OpenTelemetry.Instrumentation.EntityFrameworkCore" Version="1.0.0-beta.10" />
-    <PackageVersion Include="OpenTelemetry.Instrumentation.GrpcNetClient" Version="1.7.0-beta.1" />
-    <PackageVersion Include="OpenTelemetry.Instrumentation.Process" Version="0.5.0-beta.4" />
-    <PackageVersion Include="OpenTelemetry.Instrumentation.Quartz" Version="1.0.0-beta.1" />
-    <PackageVersion Include="OpenTelemetry.Instrumentation.Runtime" Version="1.7.0" />
-    <PackageVersion Include="OpenTelemetry.Instrumentation.SqlClient" Version="1.7.0-beta.1" />
-    <PackageVersion Include="OpenTelemetry.Instrumentation.StackExchangeRedis" Version="1.0.0-rc9.13" />
+    <PackageVersion Include="OpenTelemetry.Exporter.Prometheus.HttpListener" Version="1.8.0-rc.1" />
+    <PackageVersion Include="OpenTelemetry.Exporter.Zipkin" Version="1.8.0" />
+    <PackageVersion Include="OpenTelemetry.Extensions.Propagators" Version="1.8.0" />
+    <PackageVersion Include="OpenTelemetry.Instrumentation.AspNet" Version="1.8.0-beta.1" />
+    <PackageVersion Include="OpenTelemetry.Instrumentation.AspNetCore" Version="1.8.0" />
+    <PackageVersion Include="OpenTelemetry.Instrumentation.EntityFrameworkCore" Version="1.0.0-beta.11" />
+    <PackageVersion Include="OpenTelemetry.Instrumentation.GrpcNetClient" Version="1.8.0-beta.1" />
+    <PackageVersion Include="OpenTelemetry.Instrumentation.Process" Version="0.5.0-beta.5" />
+    <PackageVersion Include="OpenTelemetry.Instrumentation.Quartz" Version="1.0.0-beta.2" />
+    <PackageVersion Include="OpenTelemetry.Instrumentation.Runtime" Version="1.8.0" />
+    <PackageVersion Include="OpenTelemetry.Instrumentation.SqlClient" Version="1.8.0-beta.1" />
+    <PackageVersion Include="OpenTelemetry.Instrumentation.StackExchangeRedis" Version="1.0.0-rc9.14" />
     <PackageVersion Include="OpenTelemetry.Shims.OpenTracing" Version="1.7.0-beta.1" />
```

Comment:
What is your opinion about this package?
There is no changes sinse 1.7.0-beta.1 release. The only thing we can achieve is upgrade transitive dependency from OpenTelemetry.Api 1.7.0 to 1.8.0.

We ship 1.8.0 even without this change.

Reply:
I recommend shipping without any additional changes. These are not stable packages; we should rely on .NET to transitively upgrade the dependencies for these libraries from 1.7.0 to 1.8.0.

It’s a maintenance overhead for the SDK team to just modify these package dependencies to 1.8.0 and release another version.